homeassistant/components/remote/__init__.py

```diff
@@ -109,6 +112,16 @@ def is_on(hass: HomeAssistantType, entity_id: str) -> bool:
     return True
 
 
+async def async_setup_entry(hass: HomeAssistantType, entry: ConfigEntry):
```

Comment:
Return values need to be typed too.

Reply:
@balloob thanks, new to python and all these linters. i followed the code and it seems it should be a bool return type so added that.

Reply:
@balloob so its now saying:

"Returning Any from function declared to return "bool""

Any ideas is it because its awaited or something?